src/Model/SearchIndexInput.php

```diff
@@ -57,6 +57,10 @@
             throw InvalidArgumentException::invalidType('"name" option', $index['name'], 'string');
         }
 
+        if (isset($index['type']) && ! is_string($index['type'])) {
```

Comment:
This didn't come up with a local psalm check, so I assume it doesn't need to be ignored in the baseline. The error itself seems irrelevant, as the doc block typing isn't enforced at runtime.

Reply:
Any errors suppressed in the baseline are still reported on GitHub, and we dismiss these either as "false positive" or "won't fix". Since there isn't anything you can really do here, it's fine to dismiss the alert and move on :)